tests/operator/operator.go

```diff
@@ -3103,6 +3104,32 @@ func generateMigratableVMIs(num int) ([]*v1.VirtualMachineInstance, error) {
 		vmis = append(vmis, vmi)
 	}
 
+	lastVMIIndex := len(vmis) - 1
+	vmi := vmis[lastVMIIndex]
+
+	const nadName = "secondarynet"
+	netAttachDef := libnet.NewBridgeNetAttachDef(nadName, nadName)
+	_, err := libnet.CreateNetAttachDef(context.Background(), testsuite.GetTestNamespace(vmi), netAttachDef)
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
This is an odd side-effect.
The helper itself is creating VMI objects which are later created on the K8S cluster, but this part is creating a NAD as a side effect in a hidden manner.

I suggest to do this only before actually creating the VMs, even if it means calling it several times. It is probably a fit in its own setup fixture.